<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveWindow</class>
 <widget class="QWidget" name="LiveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="la_head">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../头像1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_show">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
font: 75 14pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="la_roomName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 0);
font: 75 20pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>RoomName</string>
   </property>
  </widget>
  <widget class="QLabel" name="la_hostName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);
font: 75 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/主播标志.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="la_flag">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/标志位.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_send">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>490</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 85, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>490</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_cameraFlag">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/摄像头.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lab_show_host">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_camera">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开启摄像头</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/主窗口背景.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chox_mic">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开启麦克风</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_micFlag">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/麦克风.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lab_tabFlag">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/桌面.JPG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chox_table">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>直播桌面</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chox_danmu">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹幕开</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_danmu">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images/picture.qrc">:/image/开关关.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>lab_back</zorder>
  <zorder>la_head</zorder>
  <zorder>te_show</zorder>
  <zorder>la_roomName</zorder>
  <zorder>la_hostName</zorder>
  <zorder>label</zorder>
  <zorder>la_flag</zorder>
  <zorder>le_send</zorder>
  <zorder>btn_send</zorder>
  <zorder>lab_cameraFlag</zorder>
  <zorder>lab_show_host</zorder>
  <zorder>cbox_camera</zorder>
  <zorder>chox_mic</zorder>
  <zorder>lab_micFlag</zorder>
  <zorder>lab_tabFlag</zorder>
  <zorder>chox_table</zorder>
  <zorder>chox_danmu</zorder>
  <zorder>lab_danmu</zorder>
 </widget>
 <resources>
  <include location="Images/picture.qrc"/>
 </resources>
 <connections/>
</ui>
